### PR TITLE
Improve ui of needles/caps removal

### PR DESCRIPTION
**Pull request by sloriot**
Screenshot from 2020-06-30 14-00-02

### CODE REVIEW

File: Polyhedron/demo/Polyhedron/Plugins/PMP/RemoveNeedlesDialog.ui [hunk 1]
```diff
@@ -22,15 +22,15 @@
         <string>Threshold in degrees</string>
        </property>
        <property name="title">
-        <string>Cap Threshold</string>
+        <string>Cap Threshold: Max. Angle Allowed in a Triangle</string>
```
Comment (MaelRL) [inline Polyhedron/demo/Polyhedron/Plugins/PMP/RemoveNeedlesDialog.ui:25]:
```suggestion
        <string>Cap threshold (max. angle allowed within a triangle): </string>
```

File: Polyhedron/demo/Polyhedron/Plugins/PMP/RemoveNeedlesDialog.ui [hunk 2]
```diff
@@ -57,15 +57,15 @@
         <string>(size+big edge )/(size+small edge)</string>
        </property>
        <property name="title">
-        <string>Needle Threshold</string>
+        <string>Needle Threshold:  Max. Edge Length Ratio Allowed in a Triangle (Longest/Shortest)</string>
```
Comment (MaelRL) [inline Polyhedron/demo/Polyhedron/Plugins/PMP/RemoveNeedlesDialog.ui:60]:
```suggestion
        <string>Needle threshold (max. edge length ratio (longest/shortest) allowed within a triangle):</string>
```

File: Polyhedron/demo/Polyhedron/Plugins/PMP/RemoveNeedlesDialog.ui [hunk 3]
```diff
@@ -89,18 +89,27 @@
         <string>Do not collapse edges begger than this threshold.</string>
        </property>
        <property name="title">
-        <string>Collapse Length Threshold</string>
+        <string>Needle Threshold: Maximum Length of Collapsable Edges (0 means Infinity)</string>
```
Comment (MaelRL) [inline Polyhedron/demo/Polyhedron/Plugins/PMP/RemoveNeedlesDialog.ui:92]:
```suggestion
        <string>Do not collapse needles (according to the criterion above) whose shortest edge has a length greater than:</string>
```

Comment (MaelRL) [reply Polyhedron/demo/Polyhedron/Plugins/PMP/RemoveNeedlesDialog.ui:92]: and I would put a checkbox to enable/disable (maybe for all three fields, actually?).

Comment (sloriot) [reply Polyhedron/demo/Polyhedron/Plugins/PMP/RemoveNeedlesDialog.ui]: I won't do it, it is too much work as this is not possible in the function.

**Comment by maxGimeno** [comment]
Errors : https://cgal.geometryfactory.com/CGAL/testsuite/results-5.1-Ic-188.shtml#Polyhedron_Demo

**Comment by maxGimeno** [comment]
Successfully tested in https://cgal.geometryfactory.com/CGAL/testsuite/results-5.1-Ic-190.shtml

**Pull request merged**